src/univariate/continuous/noncentralf.jl

```diff
@@ -56,8 +56,6 @@ function rand(rng::AbstractRNG, d::NoncentralF)
     r1 / r2
 end
 
-# TODO: remove RFunctions dependency once NoncentralChisq has its removed
-@rand_rdist(NoncentralF)
 function rand(d::NoncentralF)
     r1 = rand(NoncentralChisq(d.ν1,d.λ)) / d.ν1
     r2 = rand(Chisq(d.ν2)) / d.ν2
```

Comment:
```suggestion
    r2 = rand(rng, Chisq(d.ν2)) / d.ν2
```

Reply:
Don't we already have [this](https://github.com/JuliaStats/Distributions.jl/blob/master/src/univariate/continuous/noncentralf.jl#L53) ?

Reply:
Ah I hadn't noticed this since these lines were hidden in the diff. If methods with signature `rand(::AbstractRNG, ::Distribution)` already exist, we should just remove `rand(::Distribution)` completely (and check that the existing method with RNG is correct).